### PR TITLE
[DO-64] - Remove all push related code

### PR DESCRIPTION
**Pull request by ms-darianbenito**
Related to [DO-64](https://makingsense.atlassian.net/browse/DO-64)

We will create a dedicated push server for each environment.

### CODE REVIEW

**Comment by cbernat** [comment]
Are you sure you wish to remove this form here? perhaps in the future we will need to add push in Doppler right?.

File: .env [hunk 1]
```diff
@@ -14,6 +14,5 @@ REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 NODE_ICU_DATA=node_modules/full-icu
 REACT_APP_ROUTER=browser
 REACT_APP_DOPPLER_UI_LIBRARY_VERSION = v3.89.1
-REACT_APP_DATA_HUB_URL= https://hub.fromdoppler.com
```
Comment (cbernat) [inline .env:17]: Please do not remove this since we use it for datahub. (to track)

File: public/index.html [hunk 2]
```diff
@@ -52,11 +52,6 @@
       })(window,document,'script','dataLayer','GTM-TMMV2DF');
     </script>
     <!-- End Google Tag Manager -->
-    <script
```
Comment (cbernat) [inline public/index.html:55]: Please leave this as it is. We also use this to track, this is not related only to push.

Comment (ms-darianbenito) [reply public/index.html:55]: Good catch! Thanks!

**Comment by ms-darianbenito** [comment]
> Are you sure you wish to remove this form here? perhaps in the future we will need to add push in Doppler right?.

Ceci, I will talk with @hrocca about that. Thanks for the ask!

**Comment by hrocca** [comment]
> > Are you sure you wish to remove this form here? perhaps in the future we will need to add push in Doppler right?.
> 
> Ceci, I will talk with @hrocca about that. Thanks for the ask!

if necessary, we could add it again, but now we have environments for specific subscription, including production. (pushint, pushqa and pushprod in fromdoppler.net)

File: public/index.html (hunk 2 above)
Comment (ms-darianbenito) [reply public/index.html]: Done!

File: .env (hunk 1 above)
Comment (ms-darianbenito) [reply .env]: Done!

**Comment by cbernat** [comment]
👍
